docs/content/overview/introduction.md

```diff
@@ -154,30 +154,33 @@ pages.
 
 I wrote Hugo ultimately for a few reasons. First, I was disappointed with
 WordPress, my then website solution. It rendered slowly. I couldn't create
-content as efficiently as I wanted to and needed to be online to write
-posts. The constant security updates and the horror stories of people's
+content as efficiently as I wanted to and I needed to be online to write
```

Comment:
If you're going to add `I` here, you need to add a comma before `and`; however, I think the extra `I` is unnecessary.